providers/gong/metadata/schemas.json

```diff
@@ -6,11 +6,12 @@
       "objects": {
         "calls": {
           "displayName": "Calls",
-          "path": "/calls",
+          "path": "/calls/extensive",
           "responseKey": "calls",
           "fields": {
             "calendarEventId": "calendarEventId",
             "clientUniqueId": "clientUniqueId",
+            "context": "context",
```

Comment:
please remove `context` and `parties` from this list since they are not true fields.